src/zimit/zimit.py

```diff
@@ -360,6 +360,22 @@ def run(raw_args):
 
     zimit_args, warc2zim_args = parser.parse_known_args(raw_args)
 
+    logger.info("Checking browsertrix-crawler version")
+    crawl_version_cmd = ["crawl", "--version"]
+    crawl = subprocess.run(crawl_version_cmd, check=False, capture_output=True)
+    if crawl.returncode:
```

Comment:
What's the point of this? Just run with `check=True` and except on `CalledProcessError`.